packages/ui/react-ui-mosaic/src/components/Stack/virtualizer.stories.tsx

```
@@ -61,39 +62,43 @@ export const Default = {
                   width: '100%',
-                  transform: `translateY(${virtualItem.start}px)`,
                 }}
-                data-index={virtualItem.index}
-                ref={virtualizer.measureElement}
               >
-                <div className='p-1'>{virtualItem.index + 1}</div>
-                <div className='p-1'>{items[virtualItem.index].name}</div>
+                {virtualItems.map((virtualItem) => (
+                  <div
+                    key={virtualItem.key}
+                    role='list'
```

Biome (2.3.14)

[error] The elements with this role can be changed to the following elements:
<ol>
<ul>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)